renovate.json

```diff
@@ -1,5 +1,20 @@
 {
   "extends": [
-    "@nuxtjs"
+    "@nuxtjs",
+    "packages:linters",
+    ":maintainLockFilesWeekly",
+    ":enableVulnerabilityAlerts",
```

Comment:
Are these 3 relevant to automerge? (asking so if not later on move to preset)

Reply:
only `packages:linters` is relevant to automerge - and possibly would be good to get more package groups in place. The more we can group by concept the better for automerge strategy because it won't be held up by one breaking package in a long list (as with https://github.com/nuxt/modules/pull/153) - nor do we want a separate PR for every single individual update.